lib/launcher.js

```diff
@@ -20,7 +20,7 @@ var baseBrowserDecoratorFactory = function (baseLauncherDecorator, captureTimeou
   }
 }
 
-var Launcher = function (emitter, injector) {
+var Launcher = function (server, emitter, injector) {
```

Comment:
If we don't want to change the signature, I can just pull out the server from the injector directly, as I did in `lib/preprocessor.js`.